lib/core/rancher/step.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# IH_CORE_DIR will be set to the directory containing the bin and lib directories.
+
+function ih::setup::core.rancher::help() {
+  echo 'Install Rancher Desktop as an alternative to Docker Desktop'
+}
+
+# Check if the step has been installed and return 0 if it has.
+# Otherwise return 1.
+function ih::setup::core.rancher::test() {
+
+
+    # Check for PLIST FILE
+    PLISTFILE="$HOME/Library/Preferences/io.rancherdesktop.profile.defaults.plist"
+    if [ -f "$PLISTFILE" ]; then
+        return 0
+    fi
+     
+
+    ih::log::debug "Rancher Desktop is not available"
+    return 1
+}
+
+function ih::setup::core.rancher::deps() {
+  echo "core.shell"
+}
+
+
+function ih::setup::core.rancher::install() {
+    local THIS_DIR="$IH_CORE_LIB_DIR/core/rancher"
+
+    cp "${THIS_DIR}/io.rancherdesktop.profile.defaults.plist" "$HOME/Library/Preferences/io.rancherdesktop.profile.defaults.plist"
+
+
+
+    CASKSUCCEEDED=1
+    # Installation and configuration of Rancher Desktop
+    for _ in 1 2 3; do
+
+        # Detect Rosetta
+        if [[ $(sysctl -n sysctl.proc_translated) -eq 1 ]]; then
+            # Rosetta  Active
+            arch -arm64 -c brew reinstall ih-rancher
```

Comment:
I tested it. The main reason I used `reinstall` is because with just `install` the version is not upgraded. So if we already have a version installed, brew won't update the current version. I tested it in an isolated environment. 

Despite this, would you prefer to let this with just `install`?